packages/service-utils/src/cf-worker/logging.ts

```diff
@@ -0,0 +1,39 @@
+import type { ServerResponse } from "http";
+import { extractAuthorizationData } from ".";
```

Comment:
should extract this from `index.ts` as well otherwise we'll end up with cyclic dependencies